packages/backend/src/modules/tvl/ArbitrumTvl.ts

```diff
@@ -0,0 +1,56 @@
+import { ArbiscanClient, HttpClient, Logger } from '@l2beat/shared'
+import { ChainId } from '@l2beat/shared-pure'
+
+import { Config } from '../../config'
+import { BlockNumberUpdater } from '../../core/BlockNumberUpdater'
+import { Clock } from '../../core/Clock'
+import { ApplicationModule } from '../ApplicationModule'
+import { TvlDatabase } from './types'
+
+export function createArbitrumTvlModule(
+  db: TvlDatabase,
+  config: Config,
+  logger: Logger,
+  http: HttpClient,
+  clock: Clock,
+): ApplicationModule | undefined {
+  if (!config.tvl.arbitrum) {
+    logger.info('Arbitrum TVL module disabled')
+    return
+  }
+
+  // #region peripherals
+
+  const arbiscanClient = new ArbiscanClient(
+    http,
+    config.tvl.arbitrum.arbiscanApiKey,
+    logger,
+  )
+
+  // #endregion
+
+  // #region updaters
+
+  const arbiscanBlockNumberUpdater = new BlockNumberUpdater(
+    arbiscanClient,
+    db.blockNumberRepository,
+    clock,
+    logger,
+    ChainId.ETHEREUM,
```

Comment:
Double check if it shouldn't be `ChainId.ARBITRUM`

Reply:
Good catch!